scheduler/src/cook/mesos/util.clj

```diff
@@ -298,12 +300,11 @@
 ;; are set at the same time, in "real" time. This means that
 ;; jobs submitted after `start` will have been created after
 ;; expanded start
-(defn get-completed-jobs-by-user
-  "Returns all completed job entities for a particular user
-   in the specified timeframe, without a custom executor."
-  [db user start end limit state name-filter-fn]
+(defn get-jobs-by-user-and-state-and-submit
+  "Returns all jobs for a particular user in the specified timeframe, without a custom executor"
+  [db user start end state-keyword]
   (timers/time!
-    get-completed-jobs-by-user-duration
+    get-jobs-by-user-and-state-and-submit-duration
```

Comment:
Is this timer necessary? We have the necessary timers in the callers of this function.

Reply:
Removed